incidents/test_transform.py

```diff
@@ -0,0 +1,57 @@
+"""Unit tests to test transform functions"""
```

Comment:
Can add a space after the module docstring for styling purposes (and consistency with the other scripts)